test/server/rpc/procedures/utils/api-consumer.spec.js

```diff
@@ -1,81 +1,93 @@
-describe('APIConsumer', function() {
```

Comment:
I was relying on the test file name/address being printed out. This looks better but makes everything one more level nested. Done